src/app/app.module.ts

```diff
@@ -22,9 +26,27 @@ import { HttpClientModule } from '@angular/common/http';
     StoreModule.forRoot({}, {}),
     BrowserAnimationsModule,
     FormsModule,
-    LoginComponent,
+    JwtModule.forRoot({
```

Comment:
Create a providers folder and put inside this providers so this module will be cleaner.